mappings/net/minecraft/client/particle/ParticleEnchantmentTable.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS none/bmw
+CLASS none/bmw net/minecraft/client/particle/ParticleEnchantmentTable
```

Comment:
Is that the weird text symbols coming into the enchantment table as it's open? How about `Enchanting` or `EnchantSymbol` or similar?